csrc/aio/common/deepspeed_aio_common.cpp

```diff
@@ -268,6 +268,10 @@ void report_file_error(const char* filename, const std::string file_op, const in
 int open_file(const char* filename, const bool read_op)
 {
     const int flags = read_op ? (O_RDONLY | O_DIRECT) : (O_WRONLY | O_CREAT | O_DIRECT);
+#if defined(__ENABLE_CANN__)
```

Comment:
@misstek - you will want to run the pre-commit formatter on this to pass the formatting check

Reply:
Hi @loadams , sorry for the inconvenience and the formatting failure has been fixed. Would you please help me launch unit-tests again? Thank you.
```
pre-commit run --all-files
Check hooks apply to the repository......................................Passed
Check for useless excludes...............................................Passed
Check for case conflicts.................................................Passed
Check JSON...............................................................Passed
Check for broken symlinks................................................Passed
Check Yaml...............................................................Passed
Detect Destroyed Symlinks................................................Passed
Fix End of Files.........................................................Passed
fix UTF-8 byte order marker..............................................Passed
Fix python encoding pragma...............................................Passed
Mixed line ending........................................................Passed
Fix requirements.txt.....................................................Passed
Trim Trailing Whitespace.................................................Passed
yapf.....................................................................Passed
clang-format.............................................................Passed
check-torchdist..........................................................Passed
check-license............................................................Passed
codespell................................................................Passed
flake8...................................................................Passed
check-torchcuda..........................................................Passed
```